engine/evd.mli

```diff
@@ -667,9 +667,9 @@ val universes : evar_map -> UGraph.t
     [Univ.ContextSet.to_context]. *)
 val to_universe_context : evar_map -> Univ.UContext.t
 
-val univ_entry : poly:bool -> evar_map -> Entries.universes_entry
+val univ_entry : poly:bool -> evar_map -> Entries.universes_entry * UnivNames.universe_binders
```

Comment:
The pair type could be called something like `UState.named_universes_entry` (or in UnivNames, but somehow I prefer it in UState), maybe with a corresponding `val` for the empty monomorphic entry.